bfdriver/tests/test_common_load.cpp

```diff
@@ -174,4 +174,22 @@ TEST_CASE("common_load_vmm: add tss mdl fails")
     CHECK(common_fini() == BF_SUCCESS);
 }
 
+extern int platform_info_should_fail;
+
+TEST_CASE("common_load_vmm: populate_platform_info fails")
+{
+    platform_info_should_fail = 1;
+
+    binaries_info info{&g_file, g_filenames_success, false};
+
+    for (const auto &binary : info.binaries()) {
+        REQUIRE(common_add_module(binary.file, binary.file_size) == BF_SUCCESS);
+    }
+
+    CHECK(common_load_vmm() != BF_SUCCESS);
+    CHECK(common_fini() == BF_SUCCESS);
+
+    platform_info_should_fail = 0;
```

Comment:
use a gsl::finally instead as it is safer.

Reply:
oh my, that's kind of embarrassing to be honest. Yes, I should know better :P